services/app-web/src/pages/StateSubmission/ReviewSubmit/V2/ReviewSubmit/ContactsSummarySectionV2.tsx

```diff
@@ -12,15 +12,22 @@ import {
     DataDetailContactField,
 } from '../../../../../components/DataDetail'
 import { SectionCard } from '../../../../../components/SectionCard'
-import { Contract, RateRevision } from '../../../../../gen/gqlClient'
+import {
+    Contract,
+    RateRevision,
+    ContractRevision,
+} from '../../../../../gen/gqlClient'
 import {
     getDraftRates,
     getLastContractSubmission,
+    getVisibleLatestContractFormData,
 } from '../../../../../gqlHelpers/contractsAndRates'
 
 export type ContactsSummarySectionProps = {
     contract: Contract
+    contractRev?: ContractRevision
     editNavigateTo?: string
+    isStateUser: boolean
```

Comment:
nit **(no change needed now)**: To me, it feels preferable to just get the `loggedInUser` user from the `AuthContext` to access user info rather than adding more props. One of the benefits of react context is to avoid prop drilling. FYI if you ever use the auth context for the user and want to test it --  in tests you can pass in the cms or state user in the `getCurrentUserMock` to shift what user is stored there. Just make sure your test is async. 

This is a pattern I've been meaning to comment on before, wanted to share the thought forconsideration while it is fresh in my mind.